### ISSUE TITLE
Block items_spot.php

### ISSUE THREAD

**Issue by aerograf** [mambax7/publisher#10]
```
Notification: Undefined property: XoopsModules / Publisher / Item :: $ posterName in the file / modules/publisher/blocks/items_spot.php line 95
Notification: Undefined property: XoopsModules / Publisher / Item :: $ getDatesub in file / modules/publisher/blocks/items_spot.php line 95
Notification: Undefined variable: itemsCount in the file / modules/publisher/blocks/items_spot.php line 96
```

**Comment by mambax7** [comment]
Thanks!

**Comment by aerograf** [comment]
The problem arises only if you select one article to display ...

**Comment by mambax7** [comment]
Should be fixed now in 1.07 RC2
